src/components/Molecules/navigation/NavButtons.js

```diff
@@ -4,8 +4,8 @@ import './NavButtons.css';
 export const NavButtons = ({onNext, onPrevious}) => {
```

Comment:
We willen ze ook kunnen `disablen` als je niet meer verder heen of terug kan, toch? Die optie mis ik nu!